packages/common/src/services/collectibles-provider/solCollectibleHelpers.ts

```diff
@@ -467,6 +468,71 @@ const heliusNFTToCollectible = async (
   return collectible
 }
 
+const audiusBlocklistUrls = [
+  '.pro',
+  '.site',
+  '.click',
+  '.fun',
+  'sol-drift.com',
+  'myrovoucher.com',
+  'magiceden.club'
+]
+const audiusBlocklistNames = [
+  '00jup',
+  'airdrop',
+  'voucher',
+  ...audiusBlocklistUrls
+]
+export const isHeliusNFTValid = (nft: HeliusNFT, blocklist: Blocklist) => {
+  const {
+    blocklist: urlBlocklist,
+    nftBlocklist,
+    stringFilters: { nameContains, symbolContains }
+  } = blocklist
+  const {
+    grouping,
+    content: {
+      metadata: { name, symbol },
+      links: { external_url: externalUrl }
+    }
+  } = nft
+  const urlBlocklistExtended = [...urlBlocklist, ...audiusBlocklistUrls]
+  const isExternalUrlBlocked = urlBlocklistExtended.some((item) =>
```

Comment:
nit: could convert all items of blocklists to lower case with a map at the top instead of doing it inline. nbd tho.

Reply:
makes sense. can do in followup pr